src/App.css

```diff
@@ -211,3 +211,6 @@ iframe .header-container .location-btn {
 .card-sub-containers li:hover {
   scale: 1.03;
 }
+/* iframe::-webkit-scrollbar {
```

Comment:
Please remove this commented code if not required

Reply:
done